test-app/package.json

```diff
@@ -0,0 +1,118 @@
+{
+  "name": "test-app",
+  "version": "7.2.0",
```

Comment:
```suggestion
  "version": "7.2.0",
  "private": true,
```

Reply:
I added the suggestion @SergeAstapov, thank you for review!